contrib/filter-repo-demos/lint-history

```diff
@@ -73,9 +73,14 @@ EXAMPLES
     To run eslint --fix on all .js files in history:
         lint-history --relevant 'return filename.endswith(b".js")' eslint --fix
 
+    To handle the blob transformation via python code:
+        lint-history --callback '
+          blob.data=my_transform(blob.data)
+        '
+
 INTERNALS
 
-    Linting of files in history will be done by writing the files to a
+    Linting of files in history for the case of a linting programwill be done by writing the files to a
```

Comment:
"programwill" has a missing space.  But perhaps replace the new phrase with "when not using --callback"?